### PR TITLE
Mention CVSS field in template (see #248)

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ url = "https://github.com/mystuff/mycrate/issues/123"
 # "format-injection", "memory-corruption", "memory-exposure", "privilege-escalation"
 categories = ["crypto-failure"]
 
+# Optional: a Common Vulnerability Scoring System score. More information
+# can be found on the CVSS website, https://www.first.org/cvss/.
+cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
```
Comment (tarcieri) [inline README.md]: Since it's optional, I'd suggest commenting it out like the other optional fields

Comment (djc) [reply README.md]: (I took my queue from `categories`, which is also optional but not commented out.)

Comment (tarcieri) [reply README.md]: Aah, guess we aren't consistent about that. Oh well, thanks

**Comment by tarcieri** [comment]
Thanks!

**Pull request merged**